x-pack/plugins/observability/public/pages/alerts/alerts_flyout/index.tsx

```diff
@@ -58,7 +63,11 @@ export function AlertsFlyout({
   isInApp = false,
   observabilityRuleTypeRegistry,
   onClose,
+  onSelectedAlertIndexChange,
   selectedAlertId,
```

Comment:
Where do you get this from? It's optional and not passed from the alerts_table_t_grid.

Reply:
I think it's a leftover from the previous implementation of the alerts table introduced with #101632:
https://github.com/smith/kibana/blob/b307cee64e3bee0cb7bcda742c3f94f17d5d8efe/x-pack/plugins/observability/public/pages/alerts/alerts_table.tsx#L152-L157